sift/recorder/Makefile

```diff
@@ -0,0 +1 @@
+Makefile.sde
```

Comment:
We specifically removed this Makefile because we support both SDE and traditional Pin, and we select the correct one via a Makefile that is higher-up in the tree. I was thinking that we might be able to merge them, but then we could have a single Makefile that checks what to do. We don't want to compile SDE by default if we are running traditional Pin.